R/population.R

```diff
@@ -48,13 +48,19 @@ Population <- R6::R6Class(
       rClr::clrCall(self$ref, "GetCovariateValues", covariateName)
     },
     #' @description
-    #' Returns the values defined in the population for the covariate named `covariateName and invididual with id `individualId`
+    #' Returns the values defined in the population for the covariate named `covariateName` and invididual with id `individualId`
     #' @param covariateName Name of covariate for which values should be retrieved
     #' @param individualId Id of individual for which the value for covariate `covariateName` should be retrieved
     getCovariateValue = function(covariateName, individualId) {
       rClr::clrCall(self$ref, "CovariateValueFor", covariateName, as.integer(individualId))
     },
     #' @description
+    #' Returns all values defined in the population the invididual with id `individualId`
+    #' @param individualId Id of individual for which all values should be returned
+    getParameterValuesForIndividual = function(individualId) {

```

Comment:
@abdullahhamadeh New methods to retrrieve parameter values (list (paths, values)) for ONE individualId. I do not know why we would want to path more than one id and create a complex returned structure